libs/counterfactuals/src/lib/CounterfactualPanel.tsx

```diff
@@ -77,7 +77,7 @@ export class CounterfactualPanel extends React.Component<
               onChange={this.setFilterText.bind(this)}
             />
           </Stack.Item>
-          <Stack.Item>
+          <Stack.Item className={classes.detailsList}>
```

Comment:
Perhaps naming this something other than details list would be preferable since it's actually used on a Stack.